### ISSUE TITLE
Highlight TODO in comments

### ISSUE THREAD

**Issue by LaKing** [isagalaev/highlight.js#609]
It is a nice feature of gedit to highlight the word "TODO" brightly in comments.

Could this feature be added to highlight.js?

**Comment by Sannis** [comment]
@isagalaev, how about *doc-comments unification in this issue? It will be backward-incompatible with any styles that are not in highlight.js repository, but affects only few languages.

**Comment by isagalaev** [comment]
Uhm… What do you mean by "*doc-comments"?

**Comment by Sannis** [comment]
Java annotations, PHPDoc, JSDoc and other special anotations in comments, usually bold styled:

```php
/**
 * @param int $a
 * @param int $b
 * @return int
 */
function sum($a, $b) {
    return $a + $b;
}
```

**Comment by isagalaev** [comment]
Yeah, it makes sense. Let's call it "doctag" and use for everything that should be emphasized within comments. I think we can do with just a single style.

**Comment by LaKing** [comment]
Sorry for writing here, but I cant verify that feature. At least for the TODO in comments.

**Comment by isagalaev** [comment]
@LaKing could you provide an example where it doesn't work?

**Comment by LaKing** [comment]
From a fresh, unmodified git-clone, this is my processed body with some example-text.
```html
<pre><code class="js hljs javascript">
<span class="hljs-comment">/* javascript TODO FIXME NOTE BUG */</span>
x = <span class="hljs-function"><span class="hljs-keyword">function</span>(<span class="hljs-params"></span>) </span>{};
<span class="hljs-comment">// TODO</span>
<span class="hljs-comment">// FIXME</span>
<span class="hljs-comment">// NOTE</span>
<span class="hljs-comment">// BUG</span>
<span class="hljs-comment">// <span class="hljs-label">XXX - is weird!</span></span>
</code></pre> 
```
Only XXX seems to be highlighted with the label tag, .. and any comment-block-closing tag is ignored after XXX in the same line - but not for the javascript interpreters or jshint.

**Comment by isagalaev** [comment]
Apparently those special words are only highlighted when followed by a colon (`TODO:`) which was a fix for #861. 

@Sannis, we probably should use a different heuristics, like ALLCAPS, but there's a problem: case insensitivity is a global property of a language, so `/TODO/` would also match `todo` in, say, CSS. Any ideas?

**Comment by yyyc514** [comment]
So this would require changes to every theme to add a new CSS class or is there a recommendation for an existing class to match/copy?
